File: AtividadesAula/aula04/medicao_view.py
class MedicaoView:
    def exibir_tempo_rotina(self, nome_rotina: str, tempo_ms: float):
        print(f"Tempo (ms) {nome_rotina}: {tempo_ms:.4f}")

    def exibir_lista(self, lista: list, frase: str):
        print(frase)
        for item in lista:
            print(item)
        print("--------------------------")
        print(f"Total de registros: {len(lista)}")

File: AtividadesAula/aula04/MedicaoView.java
// É A CLASSE QUE CONTÉM OS PRINTS, ELA RECEBE OS DADOS PUROS E FORFMATA NA TELA 
import java.util.List;

public class MedicaoView {

    public void exibirTempoRotina(String nomeRotina, double tempoMs) {
        System.out.println("Tempo (ms) " + nomeRotina + ": " + tempoMs);
    }

    public void exibirLista(List<Integer> lista, String frase) {
        System.out.println(frase);
        for (Integer item : lista) {
            System.out.println(item);
        }
        System.out.println("--------------------------");
        System.out.println("Total de registros: " + lista.size());
    }
}
